packages/node_modules/samples/browser-plugin-meetings/app.js

```diff
@@ -1228,3 +1263,6 @@ function rejectMeeting() {
   }
 }
 
+// Separate logic for Safari enables video playback after previously
+// setting the srcObject to null regardless if autoplay is set.
+window.onload = () => addPlayIfPausedEvents(htmlMediaElements);
```

Comment:
Attaching the event handlers only once, using `onload` event to ensure the html media elements have been loaded on the page.